src/init.cpp

```diff
@@ -738,8 +742,15 @@ bool AppInit2(boost::thread_group& threadGroup)
     if (file) fclose(file);
     static boost::interprocess::file_lock lock(pathLockFile.string().c_str());
     if (!lock.try_lock())
+<<<<<<< HEAD
```

Comment:
Do not do this stuff late or early, bad things happen. Nothing to see here, moving along. Also, someone is reading these :octocat: